go.mod

```diff
@@ -1,7 +1,9 @@
-module github.com/bakatz/echo-logrusmiddleware
+module github.com/nikolay-turpitko/echo-logrusmiddleware/v2
```

Comment:
please commit this as `bakatz` instead of your username :)

Reply:
Fixed